weblab_cg/_generator_chaste_cg.py

```diff
@@ -580,107 +676,46 @@ def _get_desired_units(self, term):
     def _format_derivative_equations(self):
         """Format derivative equations for chaste output"""
         # exclude ionic currents
-        equations = []
-        d_eqs = [eq for eq in self._derivative_equations
-                 if eq not in self._default_stimulus.values()]
+#        d_eqs = [{'lhs': self._printer.doprint(eqs.lhs),
```

Comment:
Why is this here?

Reply:
Was getting ahead of myself and wrote how it would hopefully be after cellmlmanip unit conversion, but removed for now.